AllReadyApp/Web-App/AllReady/Models/FileAttachment.cs

```diff
@@ -16,10 +16,10 @@ public class FileAttachment
         public string Description { get; set; }
 
         /// <summary>MIME type of the file</summary>
-        public string MimeType { get; set; }
+        public string ContentType { get; set; }
```

Comment:
please delete this property. We can always derive content type from the file extension using FileExtensionContentTypeProvider.TryGetContentType() and passing in the path of the blog.